CONTRIBUTING.md

```diff
@@ -0,0 +1,41 @@
+# Contributing to the UOR Client and Libraries
+
+Thank you for investing your time in contributing to our project!
+
+When contributing to this repository, please first discuss the change you wish to make via issue or GitHub Discussion
+with the maintainers to ensure the change aligns with the project's long-term plans.
+
+## Pull Request Process
+
+### We Use [GitHub Flow](https://docs.github.com/en/get-started/quickstart/github-flow)
+
+Please use the following workflow to make changes to the UOR Client codebase:
+
+1. Fork the repo and create your branch from `main`.
+2. If you've added code that should be tested, add tests.
+3. If you've changed APIs, update the documentation.
+4. Ensure the test suite passes (Run `make test-unit`)
+5. Make sure your code lints (Run `make sanity`)
+6. Create a pull request against the `uor-client-go main` branch.
+
+
+When applicable, we encourage draft pull requests for early feedback and better communication.
+
+*Note: In the Makefile, there are code and file generation targets. If any changes are made to the gRPC API in the `api/services`
+directory, run `make generate-protobuf`. This operation assumes the protobuf compiler and Go plugins are installed. If any changes are made
```

Comment:
There are no targets to install some required tools, like `protoc`. Recommend adding some additional documentation on required tooling

Reply:
Added a required tools section.